docs/cosmology/index.rst

```diff
@@ -557,3 +566,4 @@ Reference/API
 =============
 
 .. automodapi:: astropy.cosmology
+
```

Comment:
Extraneous empty line? (Emacs would auto-remove it with my no-end-of-line space setting...)

Reply:
Ah. My editor has an end-of-line space setting! 😆.
I think it's PEP8 W292  (https://www.flake8rules.com/rules/W292.html)